iOS/Flows/Conversation/ConversationListCoordinator+Presentation.swift

```diff
@@ -0,0 +1,151 @@
+        query.getFirstObjectInBackground { [unowned self] object, error in
+            if let circle = object as? Circle {
+                let coordinator = CircleCoordinator(with: circle,
+                                                    router: self.router,
+                                                    deepLink: self.deepLink)
+                self.present(coordinator)
+            }
+        }
+    }
+
+    func presentThread(for channelId: ChannelId,
+                       messageId: MessageId,
+                       startingReplyId: MessageId?) {
+        
+        let coordinator = ThreadCoordinator(with: channelId,
+                                            messageId: messageId,
+                                            startingReplyId: startingReplyId,
+                                            router: self.router,
+                                            deepLink: self.deepLink)
+        self.present(coordinator)
+    }
+    
+    func presentProfilePicture() {
+        let vc = ModalPhotoViewController()
+        
+        // Because of how the People are presented, we need to properly reset the KeyboardManager.
+        vc.dismissHandlers.append { [unowned self] in
+            self.conversationListVC.becomeFirstResponder()
+        }
+        
+        vc.onDidComplete = { _ in
+            vc.dismiss(animated: true, completion: nil)
+        }
+
+        self.conversationListVC.resignFirstResponder()
+        self.router.present(vc, source: self.conversationListVC)
+    }
+
+    func presentPeoplePicker() {
+        guard let conversation = self.activeConversation else { return }
+        let coordinator = PeopleCoordinator(router: self.router, deepLink: self.deepLink)
+        self.present(coordinator) { [unowned self] people in
+            self.add(people: people, to: conversation)
+        }
+    }
+    
+    private func present<ChildResult>(_ coordinator: PresentableCoordinator<ChildResult>,
+                                      finishedHandler: ((ChildResult) -> Void)? = nil) {
+        self.removeChild()
+        
+        // Because of how the People are presented, we need to properly reset the KeyboardManager.
+        coordinator.toPresentable().dismissHandlers.append { [unowned self] in
+            self.conversationListVC.becomeFirstResponder()
+        }
+        
+        self.addChildAndStart(coordinator) { [unowned self] result in
+            self.router.dismiss(source: coordinator.toPresentable(), animated: true) { [unowned self] in
+                self.conversationListVC.becomeFirstResponder()
+                finishedHandler?(result)
+            }
+        }
+        
+        self.conversationListVC.resignFirstResponder()
+        self.router.present(coordinator, source: self.conversationListVC)
+    }
+    
+    func add(people: [Person], to conversation: Conversation) {
+        let controller = ChatClient.shared.channelController(for: conversation.cid)
```

Comment:
This may need to change based on how we want to handle invites. Do we invite to a convo or a circle? not sure...